atsign-foundation/at_client_sdk
# Exposed SyncService, NotificationService classes

**- What I did**

- Exposed SyncService, NotificationService classes
- Added minor docs for the classes.

**- How I did it**

- Forked and edited.

**- How to verify it**

- Generate the docs using `dartdoc` command and head over to the `/at_client/doc/api/at_client/AtClientManager-class.html` file and check if it has SyncService and NotificationService references.

**- Description for the changelog**
